doc/releases/release-notes-template.rst

```diff
@@ -0,0 +1,269 @@
+:orphan:
+
+.. _zephyr_X.Y:
+
+Zephyr X.Y.0 (Working Draft)
+############################
+
+We are pleased to announce the release of Zephyr version X.Y.0.
+
+.. This section is to be populated by the release manager
+
+Major features and improvements in this release include:
+
+*
+*
+
+API Changes
+***********
+
+.. Please document all API additions, removals, deprecation and changes here and
+   not in the individual subsections
+
+New APIs
+========
+
+Removed APIs
+============
+
+Deprecated APIs
+===============
+
+Changed APIs
+============
+
+.. Includes both stable and unstable/experimental API changes
+
+Security Vulnerabilities
+************************
+
+Detailed changelog
+******************
+
+Kernel
+======
+
+Architectures
+=============
+
+* ARC
+
+* ARM
+
+* ARM64
+
+* MIPS
+
+* NIOS2
+
+* POSIX
+
+* RISC-V
+
+* SPARC
+
+* x86
+
+* Xtensa
+
+Bluetooth
+=========
+
+* Audio
+
+* Direction Finding
+
+* Host
+
+* Mesh
+
+* Controller
+
+* HCI Driver
+
+Boards & SoC Support
+====================
+
+* SoC series
+
+* ARC boards
+
+* ARM boards
+
+* ARM64 boards
+
+* MIPS boards
+
+* NIOS2 boards
+
+* POSIX boards
+
+* RISC-V boards
+
+* SPARC boards
+
+* x86 boards
+
+* Xtensa boards
+
+* Other boards
```

Comment:
No need to have separate lists for socs/boards added and removed?

Reply:
I thought about that too, but I think that it's just too verbose. Instead, we can have:

```
* ARM Boards
   * Added board foo
   * Removed board bar
```